### PR TITLE
Add support for mocking Abstract classes

### PR DESCRIPTION
**Pull request by Roaders**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@morgan-stanley/ts-mocking-bird",
-    "version": "0.4.2",
+    "version": "0.6.0",
```
Comment (Davidhanson90) [inline package.json]: why the two point jump in version?

Comment (Roaders) [reply package.json]: I think I had a dead tag locally or something. Don't think it's an issue is it?

Comment (Davidhanson90) [reply package.json]: Not really but will leave an unobvious versioning gap.

**Review by Davidhanson90 — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (Roaders) [reply package.json]: fixed

**Pull request merged**